packages/types/src/payment/mutations.ts

```diff
@@ -31,17 +31,40 @@ export interface CreatePaymentDTO {
   provider_id: string
   data: Record<string, unknown>
 
+  payment_session_id: string
+  payment_collection_id: string
+
   cart_id?: string
   order_id?: string
   order_edit_id?: string
   customer_id?: string
 }
 
 export interface UpdatePaymentDTO {
+  id: string
+
   cart_id?: string
   order_id?: string
   order_edit_id?: string
   customer_id?: string
+
+  captured_at?: Date
+
+  data?: Record<string, unknown>
+}
+
+export interface CreateCaptureDTO {
+  amount: number
+  payment_id: string
+
+  captured_by?: string
+}
+
+export interface CreateRefundDTO {
+  amount: number
+  payment_id: string
+
+  captured_by?: string
```

Comment:
question: Should this be `created_by`?
```suggestion
  captured_by?: string
```